WChat/src/utils/constants.js

```diff
@@ -5,3 +5,4 @@ export const USERS_PATH = 'users';
 export const USER_PATH = 'user';
 export const GROUPS_PATH = 'groups';
 export const DEFAULT_USER = 'woloxer';
+export const BACKGROUND_IMAGE = 'https://i.imgur.com/TnNwdvV.jpg';
```

Comment:
`trucazo`, why dont you download it and put it in the assets folder?

Reply:
the reason is that im super lazy of course